src/app/components/applications/SSV/MyAccount/components/Withdraw/components/ValidatorFlow/ValidatorFlow.tsx

```diff
@@ -132,15 +134,15 @@ const ValidatorFlow = () => {
           header={'Withdraw'}
           body={secondBorderScreen}
           bottom={[
-              <Button
-                  text={buttonText}
-                  withAllowance={false}
-                  onClick={withdrawSsv}
-                  errorButton={errorButton}
-                  checkboxesText={showCheckBox ? [checkBoxText] : []}
-                  checkBoxesCallBack={showCheckBox ? [setUserAgreement] : []}
-                  disable={(clusterStore.getClusterRunWay({ ...cluster, balance: walletStore.toWei(newBalance) }) <= 30 && !userAgree) || Number(inputValue) === 0}
-              />,
+            termsConditionWrapper(<Button
+                text={buttonText}
+                withAllowance={false}
+                onClick={withdrawSsv}
+                errorButton={errorButton}
+                checkboxesText={showCheckBox ? [checkBoxText] : []}
+                checkBoxesCallBack={showCheckBox ? [setUserAgreement] : []}
+                disable={(clusterStore.getClusterRunWay({ ...cluster, balance: walletStore.toWei(newBalance) }) <= 30 && !userAgree) || Number(inputValue) === 0 || !checkedCondition}
```

Comment:
This condition is better to be calculated in component logical part and used here as final value